build.sbt

```diff
@@ -24,24 +24,25 @@ dependencyOverrides += "com.typesafe.akka" %% "akka-http-core" % akkaHttpVersion
 
 val dependencies = Seq(
   ws,
-  "uk.gov.hmrc" %% "bootstrap-play-26" % "1.3.0",
-  "uk.gov.hmrc" %% "govuk-template" % "5.52.0-play-26",
+  "uk.gov.hmrc" %% "bootstrap-frontend-play-26" % "2.24.0",
+  "uk.gov.hmrc" %% "govuk-template" % "5.56.0-play-26",
   "uk.gov.hmrc" %% "play-ui" % "8.8.0-play-26",
```

Comment:
Are there reasons for not updating play-ui & play-whitelist-filter?

Reply:
updated both versions